src/main/cpp/chassis/SwerveChassis.h

```diff
@@ -140,6 +141,11 @@ class SwerveChassis : public IChassis, public DragonDataLogger
     DragonSwervePoseEstimator *GetSwervePoseEstimator() { return m_swervePoseEstimator; }
 
 private:
+    int m_samePoseCount;
+    const int m_samePoseCountThreshold = 25;
+    const units::length::inch_t m_distanceThreshold{0.25};
+    frc::Pose2d m_prevPose;
```

Comment:
We should also initialize m_prevPose in the constructor or where we initialize m_samePoseCount currently